controllers/util/provider.go

```diff
@@ -10,17 +11,20 @@ import (
 	"encoding/json"
 	"encoding/pem"
 	"fmt"
-	"os"
 	"time"
 
 	"github.com/google/uuid"
 	"github.com/red-hat-storage/ocs-operator/v4/services"
+	corev1 "k8s.io/api/core/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+const onboardingValidationPrivateKeySecretName = "onboarding-private-key"
```

Comment:
There is no need for a const if it is only used once and is local for this file.

Reply:
The onboardingValidationPrivateKeySecretName is used at 2 places
1) onboarding-validation-key-generator 
2) provider.go

So instead of variable it is used as const.